src/test/app/SetAuth_test.cpp

```diff
@@ -46,19 +46,23 @@ struct SetAuth_test : public beast::unit_test::suite
         return jv;
     }
 
-    void testAuth(std::initializer_list<uint256> fs)
+    void testAuth(FeatureBitset features)
     {
         using namespace jtx;
         auto const gw = Account("gw");
         auto const USD = gw["USD"];
         {
-            Env env(*this, with_features(fs));
+            Env env(*this, features);
             env.fund(XRP(100000), "alice", gw);
             env(fset(gw, asfRequireAuth));
             env(auth(gw, "alice", "USD"),       ter(tecNO_LINE_REDUNDANT));
         }
         {
-            Env env(*this, with_features(featureTrustSetAuth));
+            FeatureBitset const trustSetAuth {
+                with_only_features (featureTrustSetAuth)};
+
+            Env env(*this, features | trustSetAuth);
```

Comment:
I'd remove the `trustSetAuth` var and replace `features | trustSetAuth` with `addFeatures(features, featureTrustSetAuth)`